snet-ui/Theme/theme.ts

```diff
@@ -72,6 +72,10 @@ export const lightTheme = createTheme({
     background: { default: colors.WHITE, paper: colors.WHITE },
     text: { primary: colors.BLACK, secondary: colors.WHITE_2 },
     error:{main:colors.LIGHT_RED,contrastText:colors.DARK_RED},
+    grey:{50: colors.OFF_WHITE},
```

Comment:
Don't use color names as palettes. (~grey~)

we must always use descriptive names in the palette based on their general use case.